oak_functions/client/rust/src/attestation.rs

```diff
@@ -30,7 +30,7 @@ pub fn into_server_identity_verifier(
     config_verifier: ConfigurationVerifier,
 ) -> ServerIdentityVerifier {
     let server_verifier = move |server_identity: ServerIdentity| -> anyhow::Result<()> {
-        let config = ConfigurationInfo::decode(server_identity.additional_info.as_ref())?;
+        let config = ConfigurationInfo::decode(&*server_identity.additional_info.as_slice())?;
```

Comment:
I think there should be some way of doing this without `&*`, maybe a combination of `as_bytes` , `as_slice`, `as_ref`, but I'm never sure in which order.

Reply:
yeah, I wasn't sure what the rusty convention is here. The syntax I first used was even more unwieldy: `&**server_identity.additional_info` 😅

Reply:
in steps it's basically:
deref the Arc,
get additional info of the content
get that as a slice
pass the result as a ref

Reply:
Changed to `server_identity.additional_info.as_ref().as_slice()` which seems nicer